hivemind/averaging/averager.py

```diff
@@ -138,19 +122,19 @@ def __init__(
 
         super().__init__()
         self.dht = dht
-        self.client_mode, self.listen_on, self.kwargs = client_mode, listen_on, kwargs
+        self.prefix = prefix
+
+        if client_mode is None:
+            client_mode = dht.client_mode
+        self.client_mode = client_mode
```

Comment:
Maybe rewrite to one line, like `self.client_mode if client_mode is not None else dht.client_mode`?

Reply:
I'd like to keep `client_mode` and `self.client_mode` equal in all cases (to avoid bugs when a developer accidentally used `client_mode` instead `self.client_mode` and got unexpected behavior). Therefore, one line (only the `self.client_mode` assignment) is not enough.